src/modules/utility.js

```diff
@@ -0,0 +1,92 @@
+export default class ListStore {
+  constructor() {
+    this.lists = [];
+  }
+
+  add(list) {
+    this.lists.push(list);
+  }
+
+  remove(itemId) {
+    this.lists = this.lists.filter((item) => item.id !== itemId);
+  }
+}
+
+export const updateLocalStorage = (newListStore) => {
+  localStorage.setItem('lists', JSON.stringify(newListStore.lists));
+};
+
+const tasks = document.querySelector('.tasks');
+export const renderTasks = (listInstance) => {
+  const mappedLists = listInstance.lists.map((item) => {
+    const trashBox = item.completed ? '<i class="fa-regular fa-trash-can trash-box"></i>' : '';
+    return `<li class="task-items" data-item-id="${item.id}">
+            <div class="task-list">
+                <span><i class="fa-regular fa-square unchecked"></i></span>
+                <input class="activities" type="text" value="${item.description}" >
+            </div>
+            <div class="dot-hover">
+                ${trashBox}
+                ${item.completed ? '' : '<svg class="three-dot" fill="#000000" width="20px" height="20px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><circle cx="12" cy="17.5" r="1.5" /><circle cx="12" cy="12" r="1.5" /><circle cx="12" cy="6.5" r="1.5" /></svg>'}
+            </div>
+        </li>`;
+  });
+
+  tasks.innerHTML = mappedLists.join('');
+  const taskItems = document.querySelectorAll('.task-items');
+
+  taskItems.forEach((item) => {
+    const activities = item.querySelector('.activities');
+    const dotHover = item.querySelector('.dot-hover');
+    const trashBox = dotHover.querySelector('.fa-regular.fa-trash-can');
```

Comment:
- [ ] Kindly take note that the `delete icon` is not yet displayed on the UI, I tried to hover on the the `ellipsis icon` (**the three dots**) but nothing was displayed. Kindly address this in order to have a good mastery of what's expected of you according to the project requirement. 👍